Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.1-alpine
+FROM ruby:2.6.5-alpine
```

Comment:
When I looked at updating node, I noticed that 2.6.5 image uses node 12.

Reply:
I think we should try to ensure that we are using the same version of node between what we are deploying and using locally. Perhaps we should bump node version in this pr

Reply:
sure, will give it a go